apps/api/models.py

```diff
@@ -79,7 +111,10 @@ def __unicode__(self):
 class Rubric(models.Model):
     name = models.CharField(max_length=254)
     total = models.DecimalField(max_digits=6, decimal_places=2)
-    assignment = models.ForeignKey("Assignment")
+    assignment = models.ForeignKey("Assignment", blank=True)
```

Comment:
Following Assignment comment. There is an Assignment ForeignKey here. I don’t think it is necessary with the many-to-many field in Assignment.  This seems to be the global variable issue mentioned before.